README.md

```diff
@@ -234,9 +234,11 @@ The callback changes basing on the parameters your are giving:
 3. If two parameters are given to the callback, the first will be the `error` object, the second will be the `done` callback.
 4. If three parameters are given to the callback, the first will be the `error` object, the second will be the top level `context` unless you have specified both server and override, in that case the `context` will be what the override returns, and the third the `done` callback.
 
+If no callback is provided `ready` will return a Promise that is resolved or rejected once plugins and `after` calls are completed.  No data is provided to the `.then` callback, if an error occurs it is provided to the `.catch` callback.
```

Comment:
I think we should provide the context to `resolve`.

Reply:
I've updated my working copy code to do this.  I'll update the docs and push the update once it's decided what should be returned from `app.ready(callbackForm)`.

Reply:
This has not been updated